src/ontology/uberon-edit.obo

```diff
@@ -222238,6 +222238,16 @@ synonym: "fifth molar tooth" EXACT []
 is_a: UBERON:0003655 ! molar tooth
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500127
+name: otion
+def: "The most lateral anatomical point in the mastoid region." [ISBN:9780873659505]
+comment: In cattle, red deer, and dogs this is dorsal to the opening of the external auditory meatus; in horses, camels, sheep, goats and cats this is occipital to the meatus; in pigs this is on the opening of the meatus.
```

Comment:
"this is on the opening of the meatus"... does this mean this point can exist in space or would it exist somewhere on the circumference of the opening? Is it true that across species this point is always part of the mastoid process?

"occipital to"... Is "caudal" more appropriate if you're describing anatomical orientation?

Reply:
added an online reference of the book and two "depicted by". 
I am going to delete the comment because it doesn't actually help or add anything to the definition.

Reply:
@bvarner-ebi please let me know if I added the online references correctly.